src/GraphScene.tsx

```diff
@@ -429,11 +430,13 @@ export const GraphScene: FC<GraphSceneProps & { ref?: Ref<GraphSceneRef> }> =
       );
 
       return (
-        <Fragment>
-          {nodeComponents}
-          {edgeComponents}
-          {clusterComponents}
-        </Fragment>
+        isCentered && (
+          <Fragment>
+            {nodeComponents}
+            {edgeComponents}
+            {clusterComponents}
+          </Fragment>
+        )
```

Comment:
This could make more sense named `mounted`, but that's also not exactly what it is

`isCentered` is true when the camera re-orients from it's initial z position of 1000, which happens on big graphs

Without this the label visibility calculation is usually off on the initial render

Reply:
This also gets rid of that laggy zoom out/jump on initial big graph renders which I think was previously being handled here with a `mounted && ...`